index.htlm

```diff
@@ -37,7 +38,7 @@
       <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
       <code>It is a long established fact that a reader will be distracted by the</code>
       <ul>
-        <li>using Lorem Ipsum is that it has a more-or-less </li>
+        <li>using Lorem Ipsum is that it has a more-or-less lrud sur uertps </li>
```

Comment:
- [ ] I think that you could rephrase this to be more clear and add an explanation as well so people reading your suggestion can easily understand how the code would improve. For example, it could be rephrased like this: `I think it would be better to not use dummy text and provide meaningful content. This could improve the code because ... [Add the explanation here]`. Please edit this comment in order to be clear and include the explanation! 🤓 🔧 

>This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
>
>This is a vague comment: *The README is not descriptive, why don't you make it better?*